pkg/gui/controllers/submodules_controller.go

```diff
@@ -261,6 +261,20 @@ func (self *SubmodulesController) openBulkActionsMenu() error {
 				},
 				Key: 'd',
 			},
+			{
+				LabelColumns: []string{self.c.Tr.BulkUpdateRecursiveSubmodules, style.FgRed.Sprint(self.c.Git().Submodule.BulkUpdateRecursivelyCmdObj().ToString())},
```

Comment:
I'd move this up to go right after "git submodule update".

Also, why is this red? It's not a dangerous operation, so it should be yellow like "update".